### ISSUE TITLE
Citizens Problem.

### ISSUE THREAD

**Issue by Archious** [FlyingPikachu/Quests#118]
Hi ! Im having problems with Citizens!

Myconfig:
allow-command-questing: false
allow-command-quests-with-npcs: true

But it seems like I cant right click the NPC's ..
Any solutions?

No console Error .. Its just I cant rightclick the npc .. it says " Sorry you cant use that here "

(Im using sub character to test my quests .. It doesnt work to them, they got proper quests permissions) am I missing something, player with * of all commands can use ur quest.. but if I removed * ..  cant use quests

also sometimes when I edit my config and /questsadmin reload
The config doesnt save
Im using Spigot 1.7.10

My perms for the meantime
    - quests.quest
    - quests.quit
    - quests.take
    - quests.quests
    - hsp.command.spawn
    - essentials.spawn
    - fe.member
    - essentials.afk
    - essentials.build
    - essentials.delhome
    - essentials.home
    - essentials.ignore
    - essentials.kit
    - essentials.kits
    - essentials.mail
    - essentials.mail.send
    - essentials.compass
    - essentials.motd
    - essentials.msg
    - essentials.pay
    - essentials.realname
    - essentials.reply
    - essentials.rules
    - essentials.suicide
    - essentials.sethome
    - essentials.signs.break.disposal
    - essentials.signs.create.disposal
    - essentials.signs.use.balance
    - essentials.signs.use.buy
    - essentials.signs.use.disposal
    - essentials.signs.use.heal
    - essentials.signs.use.free
    - essentials.signs.use.kit
    - essentials.signs.use.repair
    - essentials.signs.use.sell
    - essentials.signs.use.warp
    - essentials.protect.exemptbreak
    - essentials.protect.exemptusage
    - essentials.protect.exemptplacement
    - essentials.tpa
    - essentials.tpaccept
    - essentials.tpahere
    - essentials.whisper
    - essentials.list
    - essentials.msg
    - essentials.near
    - modifyworld.*

**Comment by Leescar** [comment]
Your having an issue with worldguard it sounds.
 Or some other region protection plugin
 If the npc is in a protected area of worldguard you will need to set flags on the protection region to allow USE and allow INTERACT flags.
This will allow users to use chests and other things, but not break. 
If your using another plugin I'm unsure of how to allow usage.

**Comment by Archious** [comment]
USE is allowed
I dont see anything wrong in the worldguard and region .. everything seems to be fine ..
idk what causing this ..

**Comment by FlyingPikachu** [comment]
"Sorry, but you can't _____ here" is a WorldGuard response. Make sure to check your __global__ region as well. For further support, contact the author(s) of WorldGuard or create a support ticket on their bug tracker: http://youtrack.sk89q.com/issues/WORLDGUARD

Tl;dr: Closing as this is not an issue with Quests.

**Comment by Archious** [comment]
Hey! I found out something

When I use Build deny .. Players can interact with the npc that's why the "Sorry, but you can't _____ here" is not working ..

but 

when I use Build allow .. they can interact ..
What seems to be the problem?

Looks like the citizens only fired when you allowed build ..
region is global ..

**Comment by Leescar** [comment]
Do you also have the Use flag set to allow?
I have a region where use and and interact are set to allow and they can use npcs just fine.
By what you said its an issue with worldguard behaving correctly.
So its not really a 'problem' its just a settings issue.
